### PR TITLE
Add unflattened jet/lep/V variables to array outputs; Add first GNN code

### PR DESCRIPTION
**Pull request by mondalspandan**
Crossref: https://gitlab.cern.ch/cms-analysis/hig/vhcc-run3/tasks/-/issues/9

The changes on this PR are dependent on the following change in the PoCo repo: https://github.com/PocketCoffea/PocketCoffea/pull/276

I would wait for the PoCo PR to be merged first before merging this one.

### CODE REVIEW

**Comment by mondalspandan** [comment]
With `save_arrays` set to False, I get
`ValueError: in ListOffsetArray64, cannot convert to RegularArray because subarray lengths are not regular`.
But works fine with `save_arrays` True. Need to fix this.

File: MVA/slurm/log/.gitkeep
Comment (andreypz) [inline MVA/slurm/log/.gitkeep:1]: Did you intend to commit this file?

File: MVA/slurm/submit.sh
Comment (andreypz) [inline MVA/slurm/submit.sh:1]: I this file needed on git?

**Review by andreypz — COMMENTED** [review]
This look OK, except maybe few files not needed in the repo?
Let me know when it's ready for merge

**Comment by andreypz** [comment]
The issue with `saved_arrays` will be solved in my next PR

File: MVA/slurm/submit.sh
Comment (mondalspandan) [reply MVA/slurm/submit.sh:1]: Okay I can remove this.

**Comment by mondalspandan** [comment]
> Let me know when it's ready for merge

I converted this to draft because I'm now working on the odd/even splitting and implementing a weighted loss.

Besides, we need to wait for the PocketCoffea PR to be merged. I'll let you know.

**Comment by mondalspandan** [comment]
@andreypz I added reverse compatibility with PoCo's old way of reading jets, i.e., without https://github.com/PocketCoffea/PocketCoffea/pull/276 being merged. I have not explicitly tested it, so perhaps you can confirm if it works.

So I think this can be merged now (provided you don't find issues with the jet selector). Will have to remove some temporary code when the PoCo PR gets merged.

**Review by andreypz — APPROVED** [review]

**Pull request merged**
